exercises/practice/game-of-life/.meta/example.ex

```diff
@@ -0,0 +1,44 @@
+defmodule GameOfLife do
+  @doc """
+  Apply the rules of Conway's Game of Life to a grid of cells
+  """
+
+  @spec tick(matrix :: list(list(0 | 1))) :: list(list(0 | 1))
+  def tick([]), do: []
+
+  def tick(matrix) do
+    neighbors = count_neighbors(matrix)
+
+    Enum.zip_with(matrix, neighbors, fn row_matrix, row_neighbor ->
+      Enum.zip_with(row_matrix, row_neighbor, &rule/2)
```

Comment:
Aw man, Elixir 1.11 doesn't have `Enum.zip_with`, it's the heart and soul of my algorithm :(

Reply:
I was looking up when 1.17 might come out, and noticed that 1.11 is not maintained security-wise. Maybe that's also a good cutoff for us (says the super biased contributor)

Reply:
That's a nice table! How come I never knew before that Elixir versions have a "security support" lifespan? Feel free to drop 1.11 if it's in the way, but maybe when we ad 1.17, we can keep 1.12.

Reply:
(I'm too lazy to come up with an alternative example solution to this exercise)

Reply:
> That's a nice table! How come I never knew before that Elixir versions have a "security support" lifespan? Feel free to drop 1.11 if it's in the way, but maybe when we ad 1.17, we can keep 1.12.

There is also this page: https://hexdocs.pm/elixir/compatibility-and-deprecations.html which is probably a better source.

I feel a bit conflicted about removing a whole version just for one function in an example solution. Maybe I'll simply re-define it with a comment.

Reply:
Seems like [1.17 is on its way,](https://elixirforum.com/t/elixir-v1-17-0-rc-0-released/63764) perfect timing. I'll wait for that.